sync/backup.go

```diff
@@ -71,11 +74,28 @@ func (t *Task) createBackup(replicaInController *rest.Replica, snapshot, dest, v
 
 	logrus.Infof("Backing up %s on %s, to %s", snapshot, replicaInController.Address, dest)
 
-	backup, err := repClient.CreateBackup(snapshot, dest, volumeName, labels, credential)
+	backupID, err := repClient.CreateBackup(snapshot, dest, volumeName, labels, credential)
 	if err != nil {
 		logrus.Errorf("Failed backing up %s on %s to %s", snapshot, replicaInController.Address, dest)
 		return "", err
 	}
+
+	var backup, backupErr string
+
+	for i := 0; i < BackupStatusRetryAttempts; i++ {
```

Comment:
We should convert to asynchronized process, right? And only very small backups can finish in 300 seconds.

The caller needs to change the way of making backup. It will use the backupID to query the backup status from time to time. `longhorn snapshot backup` shouldn't be a blocking call.